cerbero/build/cookbook.py

```diff
@@ -177,7 +177,8 @@ def update_step_status(self, recipe_name, step):
         @type step: str
         '''
         status = self._recipe_status(recipe_name)
-        status.steps.append(step)
+        if not step in status.steps:
+            status.steps.append(step)
```

Comment:
Didn't know about the spurious message. This is to fix the case where using the -f when building or a buildone the steps will not be added to the list of steps done (as those were already done), just an optimization.